frontend/src/components/employees/index.tsx

```diff
@@ -159,8 +164,12 @@ const Employees = () => {
 
   // Fetch data when component loads
   useEffect(() => {
-    dispatch(fetchStaff);
-  }, [dispatch]);
+    const getStaff = async () => {
+      const accessToken = await getTokenSilently();
```

Comment:
Can this fail? If so we need error handling here as well.

Reply:
If it fails it returns an undefined token, so it will just form a get request without the correct authorisation. I'm not sure if/how this method will fail unless something else has gone very wrong at Auth0 end.

I could do a check for `accessToken` being undefined or not?

Reply:
I think it's fine as it is, that was a genuine question :slightly_smiling_face: